lttoolbox-language-module/Makefile.am

```diff
@@ -77,16 +77,9 @@ apertium_{{languageCode}}_DATA=$(TARGETS_COMMON)
 pkgconfigdir = $(prefix)/share/pkgconfig
 pkgconfig_DATA = $(BASENAME).pc
 
-# Modes are not installed since they're all just "debug modes". All
-# modes are created by this goal.
-modes/$(LANG1)-morph.mode: modes.xml
```

Comment:
?

Reply:
Modes are created by `install-modes` during install - nothing else ever needs to create modes locally. This was a dead target that someone forgot to remove.